### PR TITLE
Added missing function to Stdio Serial

### PR DESCRIPTION
**Pull request by AndreasBomholtz**
Added write function with 2 parameters to Stdio Serial.

### CODE REVIEW

File: cores/epoxy/StdioSerial.h [hunk 1]
```diff
@@ -18,15 +18,16 @@ class StdioSerial: public Stream {
     void begin(unsigned long /*baud*/) { bufch = -1; }
 
     size_t write(uint8_t c) override;
+    size_t write(const uint8_t* buf, size_t len);
```
Comment (bxparks) [inline cores/epoxy/StdioSerial.h:21]: nit: Add `override` keyword.

**Review by bxparks — COMMENTED** [review]
It was not really missing, because `Print::write(buffer, size)` is defined and becomes inherited by `StdioSerial`. Of course your version is faster and cleaner, so I'll accept the PR. Just curious, did you run into performance issues?

**Comment by AndreasBomholtz** [comment]
I'm using the ArduinoJson library and it complains about the function is missing.

`ArduinoJson/src/ArduinoJson/Serialization/Writer.hpp:22:29: error: no matching function for call to ‘StdioSerial::write(const uint8_t*&, size_t&)'`
`   22 |     return _dest->write(s, n);`

So that was the reason that I needed to added it.

File: cores/epoxy/StdioSerial.h (hunk 1 above)
Comment (AndreasBomholtz) [reply cores/epoxy/StdioSerial.h]: I have added override.

**Comment by bxparks** [comment]
I discovered the underlying problem, and fixed it with 26395cb which replaces this PR. This should allow ArduinoJson to build.

The problem is an obscure (at least to me) C++ feature where an **overloaded** method in a base class prevents C++ from searching the parent classes for **overridden** methods. The `write()` function is heavily overloaded in the `Print` class. Fixed by adding a single statement `using Print::write`.

I'm going to close this PR and not incorporate your new `write(buf, n)` overridden method. Because I don't think performance is an issue, and this increases the maintenance burden of the class just a bit, and the `print(__FlashStringHelper__*)` is written as a loop over `write(c)`, and this is probably the most frequently used function, and if a loop is ok for that, it's probably ok for `write(buf, n)`.

**Comment by AndreasBomholtz** [comment]
Perfect. Thank you for fixing the issue :)

**Pull request closed without merging**